sdk/src/target/usb_device.cpp

```diff
@@ -77,3 +77,8 @@ UsbDevice::applyCalibrationToFrame(uint16_t * /*frame*/,
                                    const std::string & /*mode*/) {
     return aditof::Status::GENERIC_ERROR;
 }
+
+aditof::Status
+UsbDevice::getDetails(aditof::DeviceDetails & /*details*/) const {
```

Comment:
& /*details*/ 

Is this something I do not understand yet? or?

Reply:
This is a stub implementation so most function parameters are not used. We comment them so we don't get warnings about unused parameter. At some point we will treat warnings as errors.